grunt/tasks/schema-defaults.js

```diff
@@ -54,8 +54,11 @@ module.exports = function(grunt) {
             });
         });
 
+        var sourcedir = grunt.config("sourcedir");
```

Comment:
```js
var sourcedir = grunt.option('outputdir') || grunt.config('sourcedir');
```
?

Reply:
var sourcedir = the location of the course.json